strr-web/components/bcros/form-section/review/Subsection.vue

```diff
@@ -2,39 +2,41 @@
   <div data-test-id="form-subsection" class="bg-white p-8 mobile:px-[8px]">
     <div class="grid grid-cols-1 tablet:grid-cols-2 desktop:grid-cols-3 gap-6 desktop:mb-6">
       <BcrosFormSectionReviewItem
-        v-if="primary"
+        v-if="isPrimary || isPrimaryHostIndividual"
         :title="t('common.formLabels.hostType')"
-        :content="displayHostType[(state as PrimaryContactInformationI).contactType]"
+        :content="isPrimary
+          ? displayHostType[(state as PrimaryContactInformationI).contactType]
+          : displayHostType[HostContactTypeE.INDIVIDUAL]"
       />
       <BcrosFormSectionReviewItem
         :title="t('common.formLabels.contactName')"
-        :content="displayContactFullName(state)"
+        :content="displayContactFullName(state) || '-'"
       />
       <BcrosFormSectionReviewItem
         :title="tContact('phoneNumber')"
         :content="displayPhoneAndExt(state.phoneNumber, state.extension) || '-'"
       />
       <BcrosFormSectionReviewItem
-        v-if="!isHostTypeBusiness"
+        v-if="isPrimaryHostIndividual"
```

Comment:
The naming of this prop is a bit confusing. As it renders this for both primary and secondary contact. The use of this prop is to only render this for individual host account type and not render for business.

Reply:
When displaying the Secondary Contact in review, we need to know the type of Primary Contact (Inv or Bus). UI design has DOB, SIN etc, fields hidden based on the type of Primary Contact. Because we have dynamic fields to display, I added this explicit prop.

Reply:
I can try to rename the prop, maybe something more generic, is that the suggestion?

Reply:
Yes, the name can lead someone to wrongly assume that this would only render for primary contact and not for secondary contact.

Reply:
Renamed, please let me know

Reply:
Shouldn't it just be `isIndividualHost`?

Reply:
Hmm, well, that could be determined by checking the `formState.primaryContact.contactType` for the Primary. The Secondary Contact does not have the type in UI, we only display it but not capturing (the main radio buttons on Step 2 is for Primary).